sdk/metric/metricdata/metricdatatest/assertion_test.go

```diff
@@ -1010,3 +1013,22 @@ func TestAssertAttributesFail(t *testing.T) {
 	}
 	assert.False(t, AssertHasAttributes(fakeT, sum, attribute.Bool("A", true)))
 }
+
+func AssertMarshal[N int64 | float64](t *testing.T, expected string, i *metricdata.Extrema[N]) {
+	t.Helper()
+
+	b, err := json.Marshal(i)
+	assert.NoError(t, err)
+	assert.Equal(t, expected, string(b))
+}
+
+func TestAssertMarshal(t *testing.T) {
+	AssertMarshal(t, "-1", &minFloat64A)
```

Comment:
Missing test for empty `Extrema`. I.e.

```suggestion
	AssertMarshal(t, "", &metricdata.Extream{})
	AssertMarshal(t, "-1", &minFloat64A)
```

Reply:
empty `Extrema` test added